apps/FaceDetection/facedetection.cpp

```diff
@@ -75,7 +79,7 @@ class FaceDetector {
                        it->at(3) - it->at(1) + 2 * padding);
           cv::Mat face = frameData(rec); // take the ROI of box on the frame
           msCmp->setField(std::to_string(computeCounter), "CMP_FINISH", 0);
-          NDN_LOG_INFO("Output Queue Size: " << output->size());
+          //          NDN_LOG_INFO("Output Queue Size: " << output->size());
```

Comment:
Turn into a debug statement?

Reply:
These lines will not be there after refactoring the codes. But for now I am changing into DEBUG statement.